study-builder/studies/circadia/post-collection.conf

```diff
@@ -20,29 +20,1542 @@
   ],
   "sections": [
     {
-      "nameTemplate": null,
+      "nameTemplate":  {
+        "templateType": "TEXT",
+        "templateText": "1",
+        "variables": []
+      },
       "icons": [],
       "blocks": [
         {
-          "titleTemplate": null,
+          "titleTemplate": {
+            "templateType": "HTML",
+            "templateText": "",
+            "variables": []
+          },
           "bodyTemplate": {
             "templateType": "HTML",
-            "templateText": """<h2 class="subtitle subtitle--light">$title</h2>""",
+            "templateText": """<h2 class="activity-section-title"> $pcq_title </h2>""",
             "variables": [
               {
-                "name": "title",
+                "name": "pcq_title",
                 "translations": [
                   {
-                    "language": "en", "text": ${i18n.en.pcq_header}
+                    "language": "en", "text": ${i18n.en.pcq_title}
                   }
                 ]
               }
             ]
           },
           "blockType": "CONTENT",
           "shownExpr": null
+        },
+        {
+          "control": { include required("../snippets/picklist-question-single-list.conf") } {
+            "stableId": "BEDTIME_SLEEP_EARLIER_OR_LATER",
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": "<h3 class="question-title"> $pcq_bedtime_sleep_earlier_or_later_question </h3>"
+              "variables": [
+                {
+                  "name": "pcq_bedtime_sleep_earlier_or_later_question",
+                  "translations": [
+                    {
+                      "language": "en", "text": ${i18n.en.pcq_bedtime_sleep_earlier_or_later_question}
+                    }
+                  ]
+                }
+              ]
+            },
+            "picklistOptions": [
+              {
+                "stableId": "BEDTIME_EARLIER_YES",
```

Comment:
It's better to keep option stableIds simple. E.g. YES and NO. No need to have a question context in the answer